Packs/CommonScripts/Scripts/CreateIndicatorsFromSTIX/CreateIndicatorsFromSTIX.py

```diff
@@ -1,69 +1,71 @@
+    :return: parsed indicators in stix Parser
     """
-       Converts score (in number format) to human readable reputation format
+    if not entry_id:
+        return_error(f"Could not find file for entry id {entry_id}.")
+    comm_output = demisto.executeCommand("StixParser", {"entry_id": entry_id})
+    indicators = comm_output[0].get("Contents")
+    if is_error(comm_output[0]):
+        return_error(indicators)
+    return json.loads(indicators)
 
-       :type score: ``int``
-       :param score: The score to be formatted (required)
 
-       :return: The formatted score
-       :rtype: ``str``
-    """
-    to_str = {3: "Bad", 2: "Suspicious", 1: "Good", 0: "None"}
-    return to_str.get(score, "None")
+def create_relationships_from_entity(indicator_relationships):
+    if not indicator_relationships:
+        return None
+    entity_relationships = list()
+    for relationship in indicator_relationships:
+        relationship_object = EntityRelationship(name=relationship.get('name'),
+                                                 entity_a=relationship.get('entityA'),
+                                                 entity_b=relationship.get('entityB'),
+                                                 entity_a_type=relationship.get('entityAType'),
+                                                 entity_b_type=relationship.get('entityBType'),
+                                                 relationship_type=relationship.get('type'))
+        entity_relationships.append(relationship_object)
 
+    return entity_relationships
 
-def main():
-    args = demisto.args()
-    entry_id = args.get("entry_id", "")
-    file_path = demisto.getFilePath(entry_id).get("path")
-    if not file_path:
-        return_error("Could not find file for entry id {}.".format(entry_id))
-    with open(file_path) as file:
-        file_txt = file.read()
 
-    comm_output = demisto.executeCommand("StixParser", {"iocXml": file_txt})
-    contents = comm_output[0].get("Contents")
-    if is_error(comm_output[0]):
-        return_error(contents)
-    data = json.loads(contents)
-    indicators = [
-        {
-            "type": stix_struct_to_indicator.get(indicator.get("indicator_type")),
-            "value": indicator.get("value"),
-            "reputation": score_to_reputation(indicator.get("score")),
-            "source": indicator.get("CustomFields", {}).get("stixPackageId", "STIX Bundle"),
-            "rawJSON": indicator,
-        }
-        for indicator in data
-    ]
+def create_indicators_loop(indicators):
+    """ Create indicators using createNewIndicator automation
+
+    :param indicators: parsed indicators
+    :return: errors if exist
+    """
+    relationships_objects = list()
     errors = list()
     for indicator in indicators:
+        indicator['type'] = indicator.get('indicator_type')
+        relationship_object = create_relationships_from_entity(indicator.get('relationships'))
```

Comment:
please check with Dror if we want to add a `create_relationships` boolean arg. We do it in feed integrations, sounds legit to add it here